turbinia/evidence.py

```diff
@@ -141,14 +144,37 @@ def __init__(
     self.saved_path_type = None
 
     if self.copyable and not self.local_path:
-      raise TurbiniaException('A copyable evidence needs a source_path')
+      raise TurbiniaException(
+          '{0:s} is a copyable evidence and needs a source_path'.format(
+              self.type))
 
   def __str__(self):
     return '{0:s}:{1:s}:{2!s}'.format(self.type, self.name, self.source_path)
 
   def __repr__(self):
     return self.__str__()
 
+  @classmethod
+  def from_dict(cls, dictionary):
+    """Instanciate an Evidence object from a dictionary of attributes.
+
+    Args:
+      dictionary(dict): the attributes to set for this object.
+    Returns:
+      Evidence: the instantiated evidence.
+    """
+    name = dictionary.pop('name', None)
+    description = dictionary.pop('description', None)
+    source = dictionary.pop('source', None)
+    source_path = dictionary.pop('source_path', None)
+    tags = dictionary.pop('tag', None)
```

Comment:
Looks like this should this be `tags` instead of `tag`?

Reply:
Like, totally!